### ISSUE TITLE
Get current quota usage?

### ISSUE THREAD

**Issue by sb10** [gophercloud/gophercloud#113]
Is there a way to get the current quota usage, the equivalent of

```
$ nova absolute-limits --tenant TENANT_NAME
```

? I can use:

```
quotasets.Get(p.computeClient, os.Getenv("OS_TENANT_ID")).Extract()
```

to get the limits, but not the current usage.

**Comment by jrperritt** [comment]
> Is there a way to get the current quota usage

Not implemented yet, as far as I know.

**Comment by jtopjian** [comment]
For reference, [here's](http://developer.openstack.org/api-ref-compute-v2.1.html#limits-v2.1) the API reference for limits.

This is something I'd be interested in as well. I might be able to put this together.

**Comment by jrperritt** [comment]
closed by #121
